.changeset/fix-clickhouse-replication-local-tables.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/clickhouse': patch
+---
+
+Downgrade ClickHouse replication check on pre-existing local tables from an error to a warning log and add `allowMixedEngines: true` opt-out configuration option.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)